task-ballerina/Module.md

```diff
@@ -0,0 +1,94 @@
+## Module Overview
+
+This module provides the functionality to schedule a Ballerina job either once or periodically and to manage the execution of those jobs.
+
+### Jobs and Scheduling
+
+Every scheduling job in Ballerina needs to be represented by a Job object. Therefore, You need to create a jobs class with your custom logic to execute it when the task is triggered.
+
+The Task package has the following two scheduling systems to schedule the job:
+
+- One-time Job Execution
+- Frequency-based Job Execution 
+
+#### One-time Job Execution
+
+This API provides the functionality to schedule a job at a specified date.
+
+The following code snippet shows how to schedule a one-time job.
+
```

Comment:
Additional space.

Reply:
Removed in 71fb824a5ad4b1f99955aac7e6b2da30d82fdb61